### PR TITLE
Added and modified schemas and models pertaining to Tags

### LINKED ISSUE #4: Implement Schema and Model for Tags
Implement the schema and Model for the tags to support the functionality of having tags in the tuits. This is to ensure that the tuits are properly passed and tags are separated and stored in a separate table with proper reference.

### PR DESCRIPTION
**Pull request by Amrutha-Vempati**
Created a new schema and model for Tags. 
Modified Tuit Model and Tuit Schema to support tags

### CODE REVIEW

File: models/tags/Tags.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default interface Tags {
```
Comment (sounakbh) [inline models/tags/Tags.ts]: Can you rename the `Tags` model to `Tag` to maintain consistency?

File: models/tags/Tags.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    tag: string,
+    retags: number
```
Comment (pavithraapanch) [inline models/tags/Tags.ts]: Can we change this to "frequency", instead of "retags"?

Comment (Amrutha-Vempati) [reply models/tags/Tags.ts]: Made the changes as per comments

File: models/tags/Tags.ts (hunk 1 above)
Comment (Amrutha-Vempati) [reply models/tags/Tags.ts]: Made the changes as per comments

**Review by sounakbh — APPROVED** [review]

**Review by pavithraapanch — APPROVED** [review]
Looks good!

**Pull request merged**